template/maestrano.rb

```diff
@@ -140,7 +139,7 @@
     #
     # == Subscriptions
     # This is the list of entities (organizations,people,invoices etc.) for which you want to be
-    # notified upon creation/update in Connec!™
+    # notified upon creation/update in Connec! Copyright.
```

Comment:
TM is rather trademark than copyright, but I think you can just put nothing here, we usually don't but the TM in the code